src/content/learn/add-react-to-an-existing-project.md

```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
 <html>
   <head><title>My app</title></head>
   <body>
-    <!-- Your existing page content (in this example, it gets replaced) -->
+    <!-- Váš stávající obsah stránky (v tomto příkladu bude nahrazen) -->
   </body>
 </html>
 ```
 
 ```js index.js active
 import { createRoot } from 'react-dom/client';
 
-// Clear the existing HTML content
+// Nahradí existující obsah HTML
 document.body.innerHTML = '<div id="app"></div>';
 
-// Render your React component instead
+// Do něj nyní vykreslíme React komponentu
 const root = createRoot(document.getElementById('app'));
-root.render(<h1>Hello, world</h1>);
+root.render(<h1>Ahoj světe</h1>);
```

Comment:
Do we want to translate a text inside the code? We want to translate comments, but I am not sure about this line.

Reply:
We can decide on this later on. The Spanish translation uses `¡Hola, mundo!`, but we can keep `Hello, world`. I am okay with both.